app/controllers/foreman_puppet/api/v2/environments_controller.rb

```diff
@@ -2,7 +2,7 @@ module ForemanPuppet
   module Api
     module V2
       class EnvironmentsController < ::ForemanPuppet::Api::V2::PuppetBaseController
-        include ForemanPuppet::Api::ImportPuppetclassesCommonController
+        include ForemanPuppet::Api::ImportPuppetclassesEnvController
```

Comment:
Why are we including it here? I thought we don't want it on Environments controller?